news.html

```diff
@@ -11,6 +11,21 @@ <h1 style="margin-bottom:20px">QuTiP News</h1>
 </div>
 <br>
 
+<hr>
+
+<div class="row" style='margin-bottom: 15px'>
+    <div class="col-md-8">
+        <p style='margin-bottom: -10px'>June 7, 2021</p>
+        <h4>2021 Google Summer of Code Project Updates
+</h4>
+    </div>
+<div class="col-md-4" style='margin-top: 25px'>
+        <a href="https://felipebihaiek.medium.com/"><p>Felipe's Blog posts</p></a>
+        <a href=""><p>Purva's Blog posts</p></a>
```

Comment:
https://purva-thakre.github.io/purva-blog/

Reply:
Thank you @purva-thakre!